src/ClusterBootstrap/services/monitor/alert-manager.yaml

```diff
@@ -80,11 +80,22 @@ data:
       group_wait: 30s
       group_interval: 5m
       group_by: [alertname]
+    routes:
+    - receiver: task_user
+      group_by: [alertname, user_email]
+      match_re:
+        type: user_alert
     receivers:
     - name: "alert-email"
       email_configs:
         - to: {{ alert_info["receiver"] }}
           html: '{{ "{{" }} template "email.html" . {{ "}}" }}'
           headers:
             subject: '{{ cnf["cluster_name"] }}: {{ "{{" }} template "__subject" . {{ "}}" }}'
+    - name: "task_user"
+      email_configs:
+        - to: '{{ "{{" }} .GroupLabels.user_email {{ "}}" }}'
```

Comment:
Will there be any of our team DL in 'cc'?

Reply:
Good suggestion.